docs/release-notes.rst

```diff
@@ -3882,19 +3875,11 @@ Version 0.12.11
 
 -  Add logging to console in test mode for the Native API when using
    :class:`determined.experimental.create`.
-
 -  Improve reliability of saving checkpoints to GCS in the presence of transient network errors.
-
--  Add `an example
-   <https://github.com/determined-ai/determined/tree/master/examples/experimental/trial/unets_tf_keras>`__
```

Comment:
`https://github.com/determined-ai/determined-examples/tree/main/computer_vision/unets_tf_keras`

Reply:
++ i'll search for any remaining `/tree/master` and replace with `/tree/main`